include/z64schedule.h

```diff
@@ -28,10 +28,15 @@
  */
 
 // Macro to convert the time format used in the save struct into the format used in Schedule
-#define SCHEDULE_CONVERT_TIME(time) ((u16)((time) - 0x10000 / 360 * 90))
+#define SCHEDULE_CONVERT_TIME_ALT(time) ((time) - 0x10000 / 360 * 90)
+#define SCHEDULE_CONVERT_TIME(time) ((u16)(s32)SCHEDULE_CONVERT_TIME_ALT(time))
+
+#define SCHEDULE_CALC_TIME_ALT(hour, minute) SCHEDULE_CONVERT_TIME((((hour)*60.0f) + (minute)) * (0x10000 / 60 / 24.0f))
```

Comment:
Any reason not to just use `SCHEDULE_TIME` to match `CLOCK_TIME`?

Reply:
Because I didn't thought on that. It's a great idea